src/screencast/pipewire_screencast.c

```diff
@@ -219,6 +219,12 @@ void xdpw_pwr_enqueue_buffer(struct xdpw_screencast_instance *cast) {
 		h->dts_offset = 0;
 	}
 
+	if (cast->frame_state == XDPW_FRAME_STATE_SUCCESS) {
+		d[0].chunk->flags = SPA_CHUNK_FLAG_NONE;
+	} else {
+		d[0].chunk->flags = SPA_CHUNK_FLAG_CORRUPTED;
```

Comment:
Sweet. Can we mark the `spa_meta_header` as corrupted as well with `SPA_META_HEADER_FLAG_CORRUPTED`?